platform/viewer/src/appExtensions/MeasurementsPanel/ConnectedMeasurementTable.js

```diff
@@ -3,6 +3,7 @@ import { MeasurementTable } from '@ohif/ui';
 import OHIF from '@ohif/core';
 import moment from 'moment';
 import cornerstone from 'cornerstone-core';
+import DICOMSR from './../../lib/DICOMSR';
```

Comment:
Is there a good reason this can't live in `@ohif/core`?

Reply:
Ideally my idea was to make it work here as I can have easy access to the OHIF instance and later get it inside `@ohif/core`